examples/common/io.adb

```diff
@@ -0,0 +1,108 @@
+with Ada.Sequential_IO; Use Ada;
+with Ada.Unchecked_Conversion;
+with Ada.Text_IO; use Ada.Text_IO;
+with Ada.Exceptions; use Ada.Exceptions;
+with Ada.IO_Exceptions; use Ada.IO_Exceptions;
+
+with Interfaces.C;
+
+with GL.Types; use GL.Types;
+
+package body IO is
+
+    type tCharacter2_Array is array (1 .. Ushort'size / 8) of character;
+    type tCharacter4_Array is array (1 .. Integer'size / 8) of character;
+
+    function To_GL_UByte is new
+      Ada.Unchecked_Conversion (Character, UByte);
+    function To_GL_Integer is new
+      Ada.Unchecked_Conversion (Source => tCharacter4_Array,  Target => Int);
+    function To_GL_Float is new
+      Ada.Unchecked_Conversion (tCharacter4_Array, Single);
+    function To_GL_Short is new
+      Ada.Unchecked_Conversion (tCharacter2_Array, Ushort);
+    function To_GL_Unsigned_Integer is new
+      Ada.Unchecked_Conversion (Source => tCharacter4_Array,  Target => uint);
```

Comment:
What are these conversion functions for? You do not use most of them, and the one time you do, instantiating `Sequential_IO` with `UByte` would have been a better alternative.